2018-01-17 09:27:00.5080 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:00.9971 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:01.4514 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:01.5885 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:01.6756 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:01.7226 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:01.7842 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:01.9029 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:01.9528 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.0215 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.0511 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.0694 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.0895 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.1100 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.1100 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.1349 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.1509 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.1699 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.1959 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.1959 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.2129 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.2259 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.2359 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.2359 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.2900 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.3230 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.3290 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.3290 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.3450 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.3450 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.3630 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.3630 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.3810 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.3810 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1620 

2018-01-17 09:27:02.3940 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.3940 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.4110 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.4231 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.4231 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.4411 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.4411 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.4581 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.4691 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.4691 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.4851 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.4851 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.5041 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.5041 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.5241 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.5331 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.5331 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.5531 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.5531 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.5712 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.6062 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.6182 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.6262 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.6262 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.6262 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.6492 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.6492 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.6612 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.6612 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.6732 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.6732 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.6732 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.6902 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.6902 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.6902 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.7092 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.7092 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.7213 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.7213 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.7213 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.7403 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.7403 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.7533 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.7533 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.7533 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.7713 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.7713 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.7843 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.7843 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.7843 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.8023 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.8023 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.8233 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.8303 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.8303 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.8493 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.8493 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.8623 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.8623 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.9064 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.9164 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1680 

2018-01-17 09:27:02.9284 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.9424 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.9424 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.9594 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.9724 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.9724 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:02.9914 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.0024 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.0024 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.0205 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.0355 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.0355 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.0485 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.0485 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.0485 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.0665 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.0665 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.0785 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.0785 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.0785 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.0985 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.0985 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.1115 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.1115 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.1115 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.1295 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.1422 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.1422 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.1590 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.1590 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.1590 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.1790 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.1790 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.1930 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.1930 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.2070 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.2070 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.2230 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.2230 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.2350 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.2350 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.2350 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.2560 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.2560 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.2680 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.2680 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.2680 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.2860 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.2860 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.2976 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.2976 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.2976 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.3157 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.3157 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.3297 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.3297 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.3297 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.3507 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.3507 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.3687 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.3777 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1740 

2018-01-17 09:27:03.3777 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.4007 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.4117 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.4117 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.4317 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.4397 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.4397 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.4537 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.4537 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.4537 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.4768 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.4858 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.4858 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.5028 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.5028 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.5238 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.5348 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.5348 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.5498 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.5498 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.5628 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.5628 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.5628 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.5878 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.5878 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.6008 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.6008 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.6138 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.6309 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:03.6309 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 09:27:06.1913 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-17 09:27:06.2143 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:451 user_id:2730927 chat_id:2730927 username:Infom name: Ruslan Kasimov date: 1516130088 message:'/t @Infom 3000 @Kasim_RI' 

2018-01-17 09:27:06.8097 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /t 

2018-01-17 09:27:06.8177 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Начала парсинга транзакции в строке '@Infom 3000 @Kasim_RI' 

2018-01-17 09:27:06.8177 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь from 'Infom' 

2018-01-17 09:27:06.8288 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Сумма '3000' 

2018-01-17 09:27:06.8288 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь to 'Kasim_RI' 

2018-01-17 09:27:06.8288 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'Infom' 

2018-01-17 09:27:06.8288 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-17 09:27:06.9218 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'Kasim_RI' 

2018-01-17 09:27:06.9278 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Kasim_RI' 

2018-01-17 09:27:07.0061 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Выполнение в SQL транзакции from:Infom amount:3000 to:Kasim_RI 

2018-01-17 09:27:07.0061 | DeeBeeTeeDB.DBAPI.NewTransaction | TRACE | NewTransactionSQLParam: INSERT INTO [dbo].[transactions] ([from_user] ,[amount] ,[to_user] ,[operation_date] ,[oid]) VALUES ('Infom' ,3000 ,'Kasim_RI' ,getdate() ,0) SELECT top 1 tid FROM [dbo].[transactions] order by [operation_date] DESC 

2018-01-17 09:27:07.1467 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Kasim_RI' 

2018-01-17 09:27:07.2258 | DeeBeeTeeAlphaBot.Program.RegisterTransaction | INFO | Отправка сообщения пользователю о новой транзакции в личный чат 

2018-01-17 09:27:07.4434 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'Infom' 

2018-01-17 09:27:07.4434 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-17 09:27:07.5280 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Infom'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Infom'), 0) as Bal  

2018-01-17 09:27:07.6081 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @Infom составляет 3884,00 

2018-01-17 09:27:07.6131 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'Kasim_RI' 

2018-01-17 09:27:07.6131 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Kasim_RI' 

2018-01-17 09:27:07.7082 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Kasim_RI'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Kasim_RI'), 0) as Bal  

2018-01-17 09:27:07.7868 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @Kasim_RI составляет -3330,00 

2018-01-17 09:27:07.7868 | DeeBeeTeeDB.DBAPI.Command_transaction | INFO | Возврат результата команды transaction с сообщением Транзакция 63 успешно добавлена
Баланс пользователя @Infom составляет 3884,00
Баланс пользователя @Kasim_RI составляет -3330,00 

2018-01-17 09:27:07.9784 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-17 09:27:07.9784 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT 2730927 ,'private' ,'', 'Infom' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = 2730927) 

2018-01-17 09:27:08.0630 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-17 09:27:08.0630 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT 2730927 , 2730927, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = 2730927 AND [user_id] = 2730927) 

2018-01-17 09:27:40.1316 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1800 

2018-01-17 09:28:46.6212 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1860 

2018-01-17 09:29:51.4340 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1920 

2018-01-17 09:30:59.7767 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1980 

2018-01-17 09:32:05.9378 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2040 

2018-01-17 09:33:11.4330 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2100 

2018-01-17 09:34:16.5469 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2160 

2018-01-17 09:35:21.3451 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2220 

2018-01-17 09:36:26.2417 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2280 

2018-01-17 09:37:32.0600 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2340 

2018-01-17 09:38:39.4955 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2400 

2018-01-17 09:39:34.1927 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-17 09:39:34.1977 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:454 user_id:2730927 chat_id:2730927 username:Infom name: Ruslan Kasimov date: 1516171174 message:'/t @Kasim_RI 2200 @Infom @Kasim_RI' 

2018-01-17 09:39:34.8099 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /t 

2018-01-17 09:39:34.8199 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Начала парсинга транзакции в строке '@Kasim_RI 2200 @Infom @Kasim_RI' 

2018-01-17 09:39:34.8199 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь from 'Kasim_RI' 

2018-01-17 09:39:34.8373 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Сумма '2200' 

2018-01-17 09:39:34.8373 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Обнаружена мультитранзакция, поиск пользователей в подстроке '@Infom @Kasim_RI  

2018-01-17 09:39:34.8554 | DeeBeeTeeDB.DBAPI.Command_transaction | TRACE | add user  'Infom' 

2018-01-17 09:39:34.8554 | DeeBeeTeeDB.DBAPI.Command_transaction | TRACE | add user  'Kasim_RI' 

2018-01-17 09:39:34.8734 | DeeBeeTeeDB.DBAPI.Command_transaction | TRACE | Количество пользователей to в мультитранзакции 2, сам пользователь True 

2018-01-17 09:39:34.8822 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'Kasim_RI' 

2018-01-17 09:39:34.8822 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Kasim_RI' 

2018-01-17 09:39:34.9718 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь номер '0' to 'Infom' 

2018-01-17 09:39:34.9808 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'Infom' 

2018-01-17 09:39:34.9808 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-17 09:39:35.0727 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь номер '1' to 'Kasim_RI' 

2018-01-17 09:39:35.0727 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'Kasim_RI' 

2018-01-17 09:39:35.0909 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Kasim_RI' 

2018-01-17 09:39:35.1749 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'Kasim_RI' 

2018-01-17 09:39:35.1799 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Kasim_RI' 

2018-01-17 09:39:35.2590 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Kasim_RI'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Kasim_RI'), 0) as Bal  

2018-01-17 09:39:35.3417 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @Kasim_RI составляет -3330,00 

2018-01-17 09:39:35.3507 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Выполнение в SQL транзакции from:Kasim_RI amount:1100 to:Infom 

2018-01-17 09:39:35.3507 | DeeBeeTeeDB.DBAPI.NewTransaction | TRACE | NewTransactionSQLParam: INSERT INTO [dbo].[transactions] ([from_user] ,[amount] ,[to_user] ,[operation_date] ,[oid]) VALUES ('Kasim_RI' ,1100 ,'Infom' ,getdate() ,0) SELECT top 1 tid FROM [dbo].[transactions] order by [operation_date] DESC 

2018-01-17 09:39:35.4578 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-17 09:39:35.5402 | DeeBeeTeeAlphaBot.Program.RegisterTransaction | INFO | Отправка сообщения пользователю о новой транзакции в личный чат 

2018-01-17 09:39:35.7088 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'Infom' 

2018-01-17 09:39:35.7148 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-17 09:39:35.7952 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Infom'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Infom'), 0) as Bal  

2018-01-17 09:39:35.8738 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @Infom составляет 2784,00 

2018-01-17 09:39:35.8738 | DeeBeeTeeDB.DBAPI.Command_transaction | INFO | Возврат результата команды transaction с сообщением Обнаружена команда мультитранзакции, будет добавлено несколько транзакция с делением суммы на целые части
Транзакция 64 успешно добавлена
Баланс пользователя @Kasim_RI составляет -3330,00
Баланс пользователя @Infom составляет 2784,00
 

2018-01-17 09:39:36.0469 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-17 09:39:36.0559 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT 2730927 ,'private' ,'', 'Infom' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = 2730927) 

2018-01-17 09:39:36.1389 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-17 09:39:36.1479 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT 2730927 , 2730927, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = 2730927 AND [user_id] = 2730927) 

2018-01-17 09:39:44.7991 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2460 

2018-01-17 09:40:10.5812 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-17 09:40:10.5812 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:457 user_id:2730927 chat_id:2730927 username:Infom name: Ruslan Kasimov date: 1516171209 message:'/d' 

2018-01-17 09:40:10.6002 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /d 

2018-01-17 09:40:10.6002 | DeeBeeTeeDB.DBAPI.Command_details | DEBUG | Поиск пользователя 'Infom' 

2018-01-17 09:40:10.6172 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-17 09:40:10.7006 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Infom'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Infom'), 0) as Bal  

2018-01-17 09:40:10.7836 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | GetUserDetailsSQL: SELECT to_user, SUM(amount) as amount FROM (SELECT [tid], [from_user], [amount], [to_user] FROM [dbo].[transactions] union SELECT [tid], [to_user], [amount]*(-1), [from_user] FROM [dbo].[transactions] ) AS T WHERE from_user = 'Infom' GROUP BY to_user 

2018-01-17 09:40:10.8627 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Infom to: Kasim_RI amount: 2230,00  

2018-01-17 09:40:10.8627 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Infom to: Sergeiaf amount: 531,00  

2018-01-17 09:40:10.8807 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Infom to: Svetlana_Shulgina amount: 0,00  

2018-01-17 09:40:10.8807 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Infom to: TretyakRoman amount: 23,00  

2018-01-17 09:40:10.8947 | DeeBeeTeeDB.DBAPI.Command_details | INFO | Возврат результата команды details с сообщениемДетали пользователя @Infom. Баланс: 2784,00
 @Kasim_RI должен 2230,00 @Infom
 @Sergeiaf должен 531,00 @Infom
 @TretyakRoman должен 23,00 @Infom
 

2018-01-17 09:40:11.0636 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-17 09:40:11.0636 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT 2730927 ,'private' ,'', 'Infom' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = 2730927) 

2018-01-17 09:40:11.1568 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-17 09:40:11.1568 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT 2730927 , 2730927, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = 2730927 AND [user_id] = 2730927) 

2018-01-17 09:40:49.9308 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2520 

2018-01-17 09:41:20.1870 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-17 09:41:20.1960 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:459 user_id:2730927 chat_id:2730927 username:Infom name: Ruslan Kasimov date: 1516171279 message:'/t @Kasim_RI 1500 @Infom' 

2018-01-17 09:41:20.1960 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /t 

2018-01-17 09:41:20.2140 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Начала парсинга транзакции в строке '@Kasim_RI 1500 @Infom' 

2018-01-17 09:41:20.2140 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь from 'Kasim_RI' 

2018-01-17 09:41:20.2310 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Сумма '1500' 

2018-01-17 09:41:20.2390 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь to 'Infom' 

2018-01-17 09:41:20.2390 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'Kasim_RI' 

2018-01-17 09:41:20.2390 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Kasim_RI' 

2018-01-17 09:41:20.3280 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'Infom' 

2018-01-17 09:41:20.3330 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-17 09:41:20.4127 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Выполнение в SQL транзакции from:Kasim_RI amount:1500 to:Infom 

2018-01-17 09:41:20.4127 | DeeBeeTeeDB.DBAPI.NewTransaction | TRACE | NewTransactionSQLParam: INSERT INTO [dbo].[transactions] ([from_user] ,[amount] ,[to_user] ,[operation_date] ,[oid]) VALUES ('Kasim_RI' ,1500 ,'Infom' ,getdate() ,0) SELECT top 1 tid FROM [dbo].[transactions] order by [operation_date] DESC 

2018-01-17 09:41:20.5384 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-17 09:41:20.6207 | DeeBeeTeeAlphaBot.Program.RegisterTransaction | INFO | Отправка сообщения пользователю о новой транзакции в личный чат 

2018-01-17 09:41:20.7944 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'Kasim_RI' 

2018-01-17 09:41:20.8044 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Kasim_RI' 

2018-01-17 09:41:20.9006 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Kasim_RI'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Kasim_RI'), 0) as Bal  

2018-01-17 09:41:20.9802 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @Kasim_RI составляет -730,00 

2018-01-17 09:41:20.9802 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'Infom' 

2018-01-17 09:41:20.9889 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-17 09:41:21.0678 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Infom'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Infom'), 0) as Bal  

2018-01-17 09:41:21.1464 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @Infom составляет 1284,00 

2018-01-17 09:41:21.1464 | DeeBeeTeeDB.DBAPI.Command_transaction | INFO | Возврат результата команды transaction с сообщением Транзакция 65 успешно добавлена
Баланс пользователя @Kasim_RI составляет -730,00
Баланс пользователя @Infom составляет 1284,00 

2018-01-17 09:41:21.3297 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-17 09:41:21.3392 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT 2730927 ,'private' ,'', 'Infom' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = 2730927) 

2018-01-17 09:41:21.4225 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-17 09:41:21.4325 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT 2730927 , 2730927, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = 2730927 AND [user_id] = 2730927) 

2018-01-17 09:41:57.6149 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2580 

2018-01-17 09:43:04.0748 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2640 

2018-01-17 09:44:09.7106 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2700 

2018-01-17 09:45:14.1126 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2760 

2018-01-17 09:46:19.7615 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2820 

2018-01-17 09:47:27.2003 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2880 

2018-01-17 09:48:32.5498 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2940 

2018-01-17 09:49:38.5918 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3000 

2018-01-17 09:50:42.9193 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3060 

2018-01-17 09:51:47.3975 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3120 

2018-01-17 09:52:52.1111 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3180 

2018-01-17 09:53:56.5731 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3240 

2018-01-17 09:55:01.0843 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3300 

2018-01-17 09:56:06.2782 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3360 

2018-01-17 09:57:14.9963 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3420 

2018-01-17 09:58:22.3111 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3480 

2018-01-17 09:59:26.6544 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3540 

2018-01-17 10:00:32.7872 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3600 

2018-01-17 10:01:38.8824 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3660 

2018-01-17 10:02:46.0794 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3720 

2018-01-17 10:03:50.5995 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3780 

2018-01-17 10:04:54.9103 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3840 

2018-01-17 10:06:01.0273 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3900 

2018-01-17 10:07:07.7520 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3960 

2018-01-17 10:08:14.6567 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4020 

2018-01-17 10:09:19.5964 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4080 

2018-01-17 10:10:28.8863 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4140 

2018-01-17 10:11:35.1302 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4200 

2018-01-17 10:39:00.9544 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-17 10:39:00.9544 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:462 user_id:77473985 chat_id:77473985 username:Kasim_RI name: Roman Kasimov date: 1516173112 message:'/d' 

2018-01-17 10:39:01.5140 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /d 

2018-01-17 10:39:01.5140 | DeeBeeTeeDB.DBAPI.Command_details | DEBUG | Поиск пользователя 'Kasim_RI' 

2018-01-17 10:39:01.5340 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Kasim_RI' 

2018-01-17 10:39:01.6145 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Kasim_RI'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Kasim_RI'), 0) as Bal  

2018-01-17 10:39:01.6938 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | GetUserDetailsSQL: SELECT to_user, SUM(amount) as amount FROM (SELECT [tid], [from_user], [amount], [to_user] FROM [dbo].[transactions] union SELECT [tid], [to_user], [amount]*(-1), [from_user] FROM [dbo].[transactions] ) AS T WHERE from_user = 'Kasim_RI' GROUP BY to_user 

2018-01-17 10:39:01.7800 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Kasim_RI to: Infom amount: -730,00  

2018-01-17 10:39:01.7800 | DeeBeeTeeDB.DBAPI.Command_details | INFO | Возврат результата команды details с сообщениемДетали пользователя @Kasim_RI. Баланс: -730,00
 @Kasim_RI должен 730,00 @Infom
 

2018-01-17 10:39:01.9691 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-17 10:39:01.9691 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT 77473985 ,'private' ,'', 'Kasim_RI' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = 77473985) 

2018-01-17 10:39:02.0777 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-17 10:39:02.0777 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT 77473985 , 77473985, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = 77473985 AND [user_id] = 77473985) 

2018-01-17 10:39:02.1857 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-17 10:39:02.1857 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:463 user_id:77473985 chat_id:77473985 username:Kasim_RI name: Roman Kasimov date: 1516173130 message:'/help' 

2018-01-17 10:39:02.2046 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /help 

2018-01-17 10:39:02.2046 | DeeBeeTeeDB.DBAPI.Command_help | INFO | Возврат результата команды help с сообщением Поддерживаются команды 
/balance /b 
/details /d 
/hello 
/help 
/start 
/transaction /t 

2018-01-17 10:39:02.3881 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-17 10:39:02.3881 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT 77473985 ,'private' ,'', 'Kasim_RI' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = 77473985) 

2018-01-17 10:39:02.4811 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-17 10:39:02.4921 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT 77473985 , 77473985, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = 77473985 AND [user_id] = 77473985) 

2018-01-17 10:39:02.5771 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-17 10:39:02.5771 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:464 user_id:77473985 chat_id:77473985 username:Kasim_RI name: Roman Kasimov date: 1516173655 message:'/d' 

2018-01-17 10:39:02.5962 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /d 

2018-01-17 10:39:02.5962 | DeeBeeTeeDB.DBAPI.Command_details | DEBUG | Поиск пользователя 'Kasim_RI' 

2018-01-17 10:39:02.6142 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Kasim_RI' 

2018-01-17 10:39:02.6972 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Kasim_RI'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Kasim_RI'), 0) as Bal  

2018-01-17 10:39:02.7805 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | GetUserDetailsSQL: SELECT to_user, SUM(amount) as amount FROM (SELECT [tid], [from_user], [amount], [to_user] FROM [dbo].[transactions] union SELECT [tid], [to_user], [amount]*(-1), [from_user] FROM [dbo].[transactions] ) AS T WHERE from_user = 'Kasim_RI' GROUP BY to_user 

2018-01-17 10:39:02.8655 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Kasim_RI to: Infom amount: -730,00  

2018-01-17 10:39:02.8655 | DeeBeeTeeDB.DBAPI.Command_details | INFO | Возврат результата команды details с сообщениемДетали пользователя @Kasim_RI. Баланс: -730,00
 @Kasim_RI должен 730,00 @Infom
 

2018-01-17 10:39:03.0398 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-17 10:39:03.0488 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT 77473985 ,'private' ,'', 'Kasim_RI' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = 77473985) 

2018-01-17 10:39:03.1320 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-17 10:39:03.1410 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT 77473985 , 77473985, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = 77473985 AND [user_id] = 77473985) 

2018-01-17 10:39:03.2250 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-17 10:39:03.2340 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:465 user_id:77473985 chat_id:77473985 username:Kasim_RI name: Roman Kasimov date: 1516173659 message:'/b' 

2018-01-17 10:39:03.2340 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /b 

2018-01-17 10:39:03.2527 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'Kasim_RI' 

2018-01-17 10:39:03.2527 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Kasim_RI' 

2018-01-17 10:39:03.3556 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Kasim_RI'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Kasim_RI'), 0) as Bal  

2018-01-17 10:39:03.4394 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @Kasim_RI составляет -730,00 

2018-01-17 10:39:03.6103 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-17 10:39:03.6103 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT 77473985 ,'private' ,'', 'Kasim_RI' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = 77473985) 

2018-01-17 10:39:03.7021 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-17 10:39:03.7021 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT 77473985 , 77473985, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = 77473985 AND [user_id] = 77473985) 

2018-01-17 10:40:07.2080 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4260 

2018-01-17 10:43:17.6560 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4320 

2018-01-17 10:44:23.5192 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4380 

2018-01-17 10:44:51.6609 | DeeBeeTeeAlphaBot.Program.Main | INFO | Запуск бота 

2018-01-17 10:44:51.7020 | DeeBeeTeeAlphaBot.Program.Main | INFO | Подключение к БД 

2018-01-17 10:44:52.2604 | DeeBeeTeeAlphaBot.Program.Main | INFO | Запуск пуллинга telegram API 

2018-01-17 10:44:59.3923 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-17 10:44:59.4064 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:470 user_id:2730927 chat_id:-156558308 username:Infom name: Ruslan Kasimov date: 1516175098 message:'/b' 

2018-01-17 10:45:00.8235 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /b 

2018-01-17 10:45:00.8235 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'Infom' 

2018-01-17 10:45:00.8433 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-17 10:45:00.9247 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Infom'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Infom'), 0) as Bal  

2018-01-17 10:45:01.0073 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @Infom составляет 1284,00 

2018-01-17 10:45:01.1801 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-17 10:45:01.1901 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT -156558308 ,'group' ,'ОФФшор', 'Infom' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = -156558308) 

2018-01-17 10:45:01.2789 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-17 10:45:01.2789 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT -156558308 , 2730927, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = -156558308 AND [user_id] = 2730927) 

2018-01-17 10:45:58.3885 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 60 

2018-01-17 10:53:45.8839 | DeeBeeTeeAlphaBot.Program.Main | INFO | Запуск бота 

2018-01-17 10:53:45.9245 | DeeBeeTeeAlphaBot.Program.Main | INFO | Подключение к БД 

2018-01-17 10:53:46.5101 | DeeBeeTeeAlphaBot.Program.Main | INFO | Запуск пуллинга telegram API 

2018-01-17 10:53:46.9193 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-17 10:53:46.9343 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:473 user_id:77473985 chat_id:77473985 username:Kasim_RI name: Roman Kasimov date: 1516175428 message:'/transaction' 

2018-01-17 10:53:47.3573 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /transaction 

2018-01-17 10:53:47.3718 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Начала парсинга транзакции в строке '' 

2018-01-17 10:53:47.5450 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-17 10:53:47.5540 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT 77473985 ,'private' ,'', 'Kasim_RI' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = 77473985) 

2018-01-17 10:53:47.6375 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-17 10:53:47.6468 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT 77473985 , 77473985, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = 77473985 AND [user_id] = 77473985) 

2018-01-17 10:53:47.9009 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-17 10:53:47.9009 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:474 user_id:77473985 chat_id:77473985 username:Kasim_RI name: Roman Kasimov date: 1516175432 message:'/t' 

2018-01-17 10:53:47.9169 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /t 

2018-01-17 10:53:47.9169 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Начала парсинга транзакции в строке '' 

2018-01-17 10:53:48.1241 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-17 10:53:48.1331 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT 77473985 ,'private' ,'', 'Kasim_RI' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = 77473985) 

2018-01-17 10:53:48.2313 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-17 10:53:48.2393 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT 77473985 , 77473985, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = 77473985 AND [user_id] = 77473985) 

2018-01-17 10:53:48.5088 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-17 10:53:48.5088 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:475 user_id:288805897 chat_id:-156558308 username:TretyakRoman name: Третьяк Роман date: 1516175562 message:'/b' 

2018-01-17 10:53:48.5298 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /b 

2018-01-17 10:53:48.5418 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'TretyakRoman' 

2018-01-17 10:53:48.5418 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'TretyakRoman' 

2018-01-17 10:53:48.6277 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'TretyakRoman'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'TretyakRoman'), 0) as Bal  

2018-01-17 10:53:48.7069 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @TretyakRoman составляет -28,00 

2018-01-17 10:53:48.8813 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-17 10:53:48.8813 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT -156558308 ,'group' ,'ОФФшор', 'TretyakRoman' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = -156558308) 

2018-01-17 10:53:48.9797 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-17 10:53:48.9797 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT -156558308 , 288805897, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = -156558308 AND [user_id] = 288805897) 

2018-01-17 10:54:50.4446 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-17 10:54:50.4446 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:482 user_id:77473985 chat_id:77473985 username:Kasim_RI name: Roman Kasimov date: 1516175689 message:'/t @kasim_ri 990 @infom' 

2018-01-17 10:54:50.4676 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /t 

2018-01-17 10:54:50.4817 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Начала парсинга транзакции в строке '@kasim_ri 990 @infom' 

2018-01-17 10:54:50.4922 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь from 'kasim_ri' 

2018-01-17 10:54:50.4922 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Сумма '990' 

2018-01-17 10:54:50.4922 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь to 'infom' 

2018-01-17 10:54:50.5082 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'kasim_ri' 

2018-01-17 10:54:50.5082 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'kasim_ri' 

2018-01-17 10:54:50.5907 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'infom' 

2018-01-17 10:54:50.5907 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'infom' 

2018-01-17 10:54:50.6743 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Выполнение в SQL транзакции from:kasim_ri amount:990 to:infom 

2018-01-17 10:54:50.6846 | DeeBeeTeeDB.DBAPI.NewTransaction | TRACE | NewTransactionSQLParam: INSERT INTO [dbo].[transactions] ([from_user] ,[amount] ,[to_user] ,[operation_date] ,[oid]) VALUES ('kasim_ri' ,990 ,'infom' ,getdate() ,0) SELECT top 1 tid FROM [dbo].[transactions] order by [operation_date] DESC 

2018-01-17 10:54:50.7876 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'infom' 

2018-01-17 10:54:50.8713 | DeeBeeTeeAlphaBot.Program.RegisterTransaction | INFO | Отправка сообщения пользователю о новой транзакции в личный чат 

2018-01-17 10:54:51.0475 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'kasim_ri' 

2018-01-17 10:54:51.0545 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'kasim_ri' 

2018-01-17 10:54:51.1337 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'kasim_ri'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'kasim_ri'), 0) as Bal  

2018-01-17 10:54:51.2646 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @kasim_ri составляет 260,00 

2018-01-17 10:54:51.2646 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'infom' 

2018-01-17 10:54:51.2742 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'infom' 

2018-01-17 10:54:51.3524 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'infom'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'infom'), 0) as Bal  

2018-01-17 10:54:51.4399 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @infom составляет 294,00 

2018-01-17 10:54:51.4489 | DeeBeeTeeDB.DBAPI.Command_transaction | INFO | Возврат результата команды transaction с сообщением Транзакция 66 успешно добавлена
Баланс пользователя @kasim_ri составляет 260,00
Баланс пользователя @infom составляет 294,00 

2018-01-17 10:54:51.6388 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-17 10:54:51.6468 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT 77473985 ,'private' ,'', 'Kasim_RI' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = 77473985) 

2018-01-17 10:54:51.7301 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-17 10:54:51.7391 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT 77473985 , 77473985, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = 77473985 AND [user_id] = 77473985) 

2018-01-17 10:54:51.9987 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 60 

2018-01-17 10:55:15.6688 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-17 10:55:15.6793 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:486 user_id:77473985 chat_id:77473985 username:Kasim_RI name: Roman Kasimov date: 1516175715 message:'/t @kasim_ri 1200 @infom' 

2018-01-17 10:55:15.6793 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /t 

2018-01-17 10:55:15.6993 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Начала парсинга транзакции в строке '@kasim_ri 1200 @infom' 

2018-01-17 10:55:15.6993 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь from 'kasim_ri' 

2018-01-17 10:55:15.7121 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Сумма '1200' 

2018-01-17 10:55:15.7121 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь to 'infom' 

2018-01-17 10:55:15.7249 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'kasim_ri' 

2018-01-17 10:55:15.7249 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'kasim_ri' 

2018-01-17 10:55:15.8094 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'infom' 

2018-01-17 10:55:15.8094 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'infom' 

2018-01-17 10:55:15.8956 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Выполнение в SQL транзакции from:kasim_ri amount:1200 to:infom 

2018-01-17 10:55:15.8956 | DeeBeeTeeDB.DBAPI.NewTransaction | TRACE | NewTransactionSQLParam: INSERT INTO [dbo].[transactions] ([from_user] ,[amount] ,[to_user] ,[operation_date] ,[oid]) VALUES ('kasim_ri' ,1200 ,'infom' ,getdate() ,0) SELECT top 1 tid FROM [dbo].[transactions] order by [operation_date] DESC 

2018-01-17 10:55:16.0011 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'infom' 

2018-01-17 10:55:16.0795 | DeeBeeTeeAlphaBot.Program.RegisterTransaction | INFO | Отправка сообщения пользователю о новой транзакции в личный чат 

2018-01-17 10:55:16.2475 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'kasim_ri' 

2018-01-17 10:55:16.2475 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'kasim_ri' 

2018-01-17 10:55:16.3372 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'kasim_ri'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'kasim_ri'), 0) as Bal  

2018-01-17 10:55:16.4161 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @kasim_ri составляет 1460,00 

2018-01-17 10:55:16.4161 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'infom' 

2018-01-17 10:55:16.4261 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'infom' 

2018-01-17 10:55:16.5042 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'infom'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'infom'), 0) as Bal  

2018-01-17 10:55:16.5825 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @infom составляет -906,00 

2018-01-17 10:55:16.5825 | DeeBeeTeeDB.DBAPI.Command_transaction | INFO | Возврат результата команды transaction с сообщением Транзакция 67 успешно добавлена
Баланс пользователя @kasim_ri составляет 1460,00
Баланс пользователя @infom составляет -906,00 

2018-01-17 10:55:16.7781 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-17 10:55:16.7781 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT 77473985 ,'private' ,'', 'Kasim_RI' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = 77473985) 

2018-01-17 10:55:16.8628 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-17 10:55:16.8679 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT 77473985 , 77473985, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = 77473985 AND [user_id] = 77473985) 

2018-01-17 10:55:56.9171 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 120 

2018-01-17 10:57:02.3299 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 180 

2018-01-17 10:58:08.3702 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 240 

2018-01-17 10:59:12.8957 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 300 

2018-01-17 11:00:17.2928 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 360 

2018-01-17 11:01:24.8018 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 420 

2018-01-17 11:02:30.2665 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 480 

2018-01-17 11:03:34.8354 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 540 

2018-01-17 11:04:41.9332 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 600 

2018-01-17 11:05:46.5443 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 660 

2018-01-17 11:06:51.1460 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 720 

2018-01-17 11:07:56.0550 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 780 

2018-01-17 11:09:01.0071 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 840 

2018-01-17 11:10:07.3524 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 900 

2018-01-17 11:11:12.1569 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 960 

2018-01-17 11:12:19.6784 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1020 

2018-01-17 11:13:24.0508 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1080 

2018-01-17 11:14:28.6134 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1140 

2018-01-17 11:15:33.1973 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1200 

2018-01-17 11:16:38.2316 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1260 

2018-01-17 11:17:45.2602 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1320 

2018-01-17 11:18:51.2769 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1380 

2018-01-17 11:19:56.0672 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1440 

2018-01-17 11:21:05.3651 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1500 

2018-01-17 11:22:13.4484 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1560 

2018-01-17 11:23:18.1374 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1620 

2018-01-17 11:24:22.8857 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1680 

2018-01-17 11:25:27.7444 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1740 

2018-01-17 11:26:32.7694 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1800 

2018-01-17 11:27:37.8552 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1860 

2018-01-17 11:28:43.1009 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1920 

2018-01-17 11:29:51.0167 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1980 

2018-01-17 11:30:55.6375 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2040 

2018-01-17 11:32:03.6639 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2100 

2018-01-17 11:33:08.1105 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2160 

2018-01-17 11:34:12.6482 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2220 

2018-01-17 11:35:17.2875 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2280 

2018-01-17 11:36:21.7909 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2340 

2018-01-17 11:37:26.3374 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2400 

2018-01-17 11:38:33.1652 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2460 

2018-01-17 11:39:37.9120 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2520 

2018-01-17 11:40:42.1938 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2580 

2018-01-17 11:41:49.8224 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2640 

2018-01-17 11:43:16.6050 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2700 

2018-01-17 11:44:23.2233 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2760 

2018-01-17 11:45:31.7878 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2820 

2018-01-17 11:46:38.8105 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2880 

2018-01-17 11:47:44.6512 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2940 

2018-01-17 11:48:51.8236 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3000 

2018-01-17 11:49:57.6580 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3060 

2018-01-17 11:51:02.5119 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3120 

2018-01-17 11:52:07.5261 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3180 

2018-01-17 11:53:14.2817 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3240 

2018-01-17 11:54:18.8702 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3300 

2018-01-17 11:55:25.7935 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3360 

2018-01-17 11:56:31.1462 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3420 

2018-01-17 11:57:37.7655 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3480 

2018-01-17 11:58:45.6688 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3540 

2018-01-17 11:59:57.2345 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3600 

2018-01-17 12:01:06.1598 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3660 

2018-01-17 12:02:12.8276 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3720 

2018-01-17 12:03:19.6950 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3780 

2018-01-17 12:04:25.0255 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3840 

2018-01-17 12:05:29.7712 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3900 

2018-01-17 12:06:34.2445 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3960 

2018-01-17 12:07:42.2045 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4020 

2018-01-17 12:08:50.8797 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4080 

2018-01-17 12:09:59.3058 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4140 

2018-01-17 12:11:06.5230 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4200 

2018-01-17 12:12:13.5177 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4260 

2018-01-17 12:13:18.0414 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4320 

2018-01-17 12:14:22.3845 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4380 

2018-01-17 12:15:31.0269 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4440 

2018-01-17 12:16:37.2475 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4500 

2018-01-17 12:17:43.5580 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4560 

2018-01-17 12:18:53.4024 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4620 

2018-01-17 12:19:59.6513 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4680 

2018-01-17 12:21:09.1694 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4740 

2018-01-17 12:22:16.1257 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4800 

2018-01-17 12:23:20.9259 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4860 

2018-01-17 12:24:25.8062 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4920 

2018-01-17 12:25:30.6273 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4980 

2018-01-17 12:26:38.8730 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5040 

2018-01-17 12:27:44.5415 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5100 

2018-01-17 12:28:49.2143 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5160 

2018-01-17 12:29:57.4749 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5220 

2018-01-17 12:31:02.5015 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5280 

2018-01-17 12:32:08.1879 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5340 

2018-01-17 12:33:13.2945 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5400 

2018-01-17 12:34:18.0065 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5460 

2018-01-17 12:35:22.5963 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5520 

2018-01-17 12:36:27.0125 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5580 

2018-01-17 12:37:31.5653 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5640 

2018-01-17 12:38:36.4415 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5700 

2018-01-17 12:39:41.4886 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5760 

2018-01-17 12:40:46.1086 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5820 

2018-01-17 12:41:50.5130 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5880 

2018-01-17 12:43:12.9238 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5940 

2018-01-17 12:44:18.1975 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6000 

2018-01-17 12:45:24.3397 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6060 

2018-01-17 12:46:31.0199 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6120 

2018-01-17 12:47:35.4015 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6180 

2018-01-17 12:48:41.9798 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6240 

2018-01-17 12:49:46.4701 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6300 

2018-01-17 12:50:53.4458 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6360 

2018-01-17 12:51:58.0752 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6420 

2018-01-17 12:53:04.9266 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6480 

2018-01-17 12:54:09.4293 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6540 

2018-01-17 12:55:14.0028 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6600 

2018-01-17 12:56:18.4042 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6660 

2018-01-17 12:57:24.6719 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6720 

2018-01-17 12:58:32.9447 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6780 

2018-01-17 12:59:42.0021 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6840 

2018-01-17 13:00:52.1592 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6900 

2018-01-17 13:02:01.3003 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6960 

2018-01-17 13:03:08.0414 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7020 

2018-01-17 13:04:13.3995 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7080 

2018-01-17 13:05:18.8817 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7140 

2018-01-17 13:06:25.9056 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7200 

2018-01-17 13:07:30.3770 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7260 

2018-01-17 13:08:35.1345 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7320 

2018-01-17 13:09:39.5453 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7380 

2018-01-17 13:10:44.9101 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7440 

2018-01-17 13:11:49.4211 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7500 

2018-01-17 13:12:54.4613 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7560 

2018-01-17 13:13:59.8062 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7620 

2018-01-17 13:15:04.5152 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7680 

2018-01-17 13:16:10.8498 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7740 

2018-01-17 13:17:16.9873 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7800 

2018-01-17 13:18:21.6581 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7860 

2018-01-17 13:19:29.0753 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7920 

2018-01-17 13:20:35.1353 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7980 

2018-01-17 13:21:40.8014 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8040 

2018-01-17 13:22:45.4904 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8100 

2018-01-17 13:23:49.8527 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8160 

2018-01-17 13:24:54.4246 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8220 

2018-01-17 13:26:01.0938 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8280 

2018-01-17 13:27:05.6155 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8340 

2018-01-17 13:28:16.2061 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8400 

2018-01-17 13:29:20.6965 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8460 

2018-01-17 13:30:26.2960 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8520 

2018-01-17 13:31:32.1710 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8580 

2018-01-17 13:32:38.6393 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8640 

2018-01-17 13:33:43.1472 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8700 

2018-01-17 13:34:50.9807 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8760 

2018-01-17 14:29:16.6859 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:17.0370 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:17.4773 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:17.7915 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:17.8716 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:17.9136 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:17.9476 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:17.9586 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:17.9706 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:17.9706 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:17.9996 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.0057 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.0057 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.0227 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.0227 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.0367 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.0367 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.0497 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.0497 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.0647 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.0797 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.0797 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.1027 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.1147 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.1147 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.1267 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8820 

2018-01-17 14:29:18.1267 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.1267 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.1427 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.1427 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.1427 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.1608 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.1608 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.1608 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.1788 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.1788 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.1918 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.1918 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.1918 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.2058 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.2058 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.2058 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.2208 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.2208 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.2208 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.2208 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.2418 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.2728 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.2858 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.2858 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.2988 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.2988 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.2988 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.3179 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.3179 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.3309 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.3309 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.3449 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.3449 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.3449 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.3619 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.3619 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.3619 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.3799 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.3799 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.3799 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.3959 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.3959 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.3959 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.4109 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.4109 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.4109 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.4269 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.4269 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.4269 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.4429 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.4429 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.4429 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.4570 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.4570 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.4570 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.4720 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.4720 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.4720 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.4860 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.4860 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.4860 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8880 

2018-01-17 14:29:18.5010 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.5010 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.5010 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.5170 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.5170 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.5170 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.5340 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.5980 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.5980 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.6171 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.6281 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.6281 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.6281 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.6511 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.6991 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.7151 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.7381 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.7381 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.7531 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.7531 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.7531 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.7692 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.7692 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.7692 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.7872 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.7872 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8002 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8002 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8002 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8162 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8162 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8302 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8302 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8302 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8462 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8462 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8462 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8612 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8612 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8612 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8762 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8762 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8762 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8762 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8952 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8952 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.8952 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.9103 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.9103 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.9103 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.9243 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.9243 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.9243 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.9393 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.9393 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.9393 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.9393 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.9583 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.9583 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.9583 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.9733 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8940 

2018-01-17 14:29:18.9733 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.9733 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.9903 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:18.9903 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:19.0013 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:19.0013 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:19.0013 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:19.0193 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:19.0193 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:19.0343 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:19.0343 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:19.0483 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:19.0483 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:19.0664 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:19.0664 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:19.0804 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:19.0804 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:19.1011 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:19.1106 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:19.1106 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:19.1106 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:19.1318 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:29:19.1318 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 14:30:02.4586 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9000 

2018-01-17 14:31:12.9418 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9060 

2018-01-17 14:32:02.0900 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-17 14:32:02.0900 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:490 user_id:459569 chat_id:-156558308 username:Sergeiaf name: Сергей Федоров date: 1516188721 message:'/t @Sergeiaf  1400 @Infom @Infom  @TretyakRoman @Sergeiaf' 

2018-01-17 14:32:02.6622 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /t 

2018-01-17 14:32:02.6622 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Начала парсинга транзакции в строке '@Sergeiaf  1400 @Infom @Infom  @TretyakRoman @Sergeiaf' 

2018-01-17 14:32:02.6802 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь from 'Sergeiaf' 

2018-01-17 14:32:02.6802 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Сумма '1400' 

2018-01-17 14:32:02.6973 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Обнаружена мультитранзакция, поиск пользователей в подстроке '@Infom @Infom  @TretyakRoman @Sergeiaf  

2018-01-17 14:32:02.6973 | DeeBeeTeeDB.DBAPI.Command_transaction | TRACE | add user  'Infom' 

2018-01-17 14:32:02.7153 | DeeBeeTeeDB.DBAPI.Command_transaction | TRACE | add user  'Infom' 

2018-01-17 14:32:02.7223 | DeeBeeTeeDB.DBAPI.Command_transaction | TRACE | add user  'TretyakRoman' 

2018-01-17 14:32:02.7223 | DeeBeeTeeDB.DBAPI.Command_transaction | TRACE | add user  'Sergeiaf' 

2018-01-17 14:32:02.7223 | DeeBeeTeeDB.DBAPI.Command_transaction | TRACE | Количество пользователей to в мультитранзакции 4, сам пользователь True 

2018-01-17 14:32:02.7383 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'Sergeiaf' 

2018-01-17 14:32:02.7383 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Sergeiaf' 

2018-01-17 14:32:02.9515 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь номер '0' to 'Infom' 

2018-01-17 14:32:02.9515 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'Infom' 

2018-01-17 14:32:02.9605 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-17 14:32:03.0382 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь номер '1' to 'Infom' 

2018-01-17 14:32:03.0382 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'Infom' 

2018-01-17 14:32:03.0382 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-17 14:32:03.1297 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь номер '2' to 'TretyakRoman' 

2018-01-17 14:32:03.1297 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'TretyakRoman' 

2018-01-17 14:32:03.1297 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'TretyakRoman' 

2018-01-17 14:32:03.2171 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь номер '3' to 'Sergeiaf' 

2018-01-17 14:32:03.2171 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'Sergeiaf' 

2018-01-17 14:32:03.2281 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Sergeiaf' 

2018-01-17 14:32:03.3073 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'Sergeiaf' 

2018-01-17 14:32:03.3073 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Sergeiaf' 

2018-01-17 14:32:03.3904 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Sergeiaf'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Sergeiaf'), 0) as Bal  

2018-01-17 14:32:03.4719 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @Sergeiaf составляет -526,00 

2018-01-17 14:32:03.4769 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Выполнение в SQL транзакции from:Sergeiaf amount:350 to:Infom 

2018-01-17 14:32:03.4769 | DeeBeeTeeDB.DBAPI.NewTransaction | TRACE | NewTransactionSQLParam: INSERT INTO [dbo].[transactions] ([from_user] ,[amount] ,[to_user] ,[operation_date] ,[oid]) VALUES ('Sergeiaf' ,350 ,'Infom' ,getdate() ,0) SELECT top 1 tid FROM [dbo].[transactions] order by [operation_date] DESC 

2018-01-17 14:32:03.6050 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-17 14:32:03.6839 | DeeBeeTeeAlphaBot.Program.RegisterTransaction | INFO | Отправка сообщения пользователю о новой транзакции в личный чат 

2018-01-17 14:32:03.8760 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'Infom' 

2018-01-17 14:32:03.8851 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-17 14:32:03.9682 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Infom'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Infom'), 0) as Bal  

2018-01-17 14:32:04.0484 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @Infom составляет 934,00 

2018-01-17 14:32:04.0524 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Выполнение в SQL транзакции from:Sergeiaf amount:350 to:Infom 

2018-01-17 14:32:04.0524 | DeeBeeTeeDB.DBAPI.NewTransaction | TRACE | NewTransactionSQLParam: INSERT INTO [dbo].[transactions] ([from_user] ,[amount] ,[to_user] ,[operation_date] ,[oid]) VALUES ('Sergeiaf' ,350 ,'Infom' ,getdate() ,0) SELECT top 1 tid FROM [dbo].[transactions] order by [operation_date] DESC 

2018-01-17 14:32:04.1465 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-17 14:32:04.2255 | DeeBeeTeeAlphaBot.Program.RegisterTransaction | INFO | Отправка сообщения пользователю о новой транзакции в личный чат 

2018-01-17 14:32:04.4007 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'Infom' 

2018-01-17 14:32:04.4107 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-17 14:32:04.4931 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Infom'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Infom'), 0) as Bal  

2018-01-17 14:32:04.5760 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @Infom составляет 584,00 

2018-01-17 14:32:04.5760 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Выполнение в SQL транзакции from:Sergeiaf amount:350 to:TretyakRoman 

2018-01-17 14:32:04.5870 | DeeBeeTeeDB.DBAPI.NewTransaction | TRACE | NewTransactionSQLParam: INSERT INTO [dbo].[transactions] ([from_user] ,[amount] ,[to_user] ,[operation_date] ,[oid]) VALUES ('Sergeiaf' ,350 ,'TretyakRoman' ,getdate() ,0) SELECT top 1 tid FROM [dbo].[transactions] order by [operation_date] DESC 

2018-01-17 14:32:04.7074 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'TretyakRoman' 

2018-01-17 14:32:04.7910 | DeeBeeTeeAlphaBot.Program.RegisterTransaction | INFO | Отправка сообщения пользователю о новой транзакции в личный чат 

2018-01-17 14:32:04.9621 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Удаленный сервер возвратил ошибку: (403) Запрещено. 

2018-01-17 14:32:21.8886 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9120 

2018-01-17 14:33:27.7703 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9180 

2018-01-17 14:34:40.0228 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9240 

2018-01-17 14:35:44.3975 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9300 

2018-01-17 14:36:16.8884 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-17 14:36:16.8934 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:493 user_id:2730927 chat_id:-156558308 username:Infom name: Ruslan Kasimov date: 1516188976 message:'/b@DeeBeeTeeBot' 

2018-01-17 14:36:17.3643 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /b 

2018-01-17 14:36:17.3643 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'Infom' 

2018-01-17 14:36:17.3793 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-17 14:36:17.4595 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Infom'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Infom'), 0) as Bal  

2018-01-17 14:36:17.5423 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @Infom составляет 584,00 

2018-01-17 14:36:17.7278 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-17 14:36:17.7368 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT -156558308 ,'group' ,'ОФФшор', 'Infom' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = -156558308) 

2018-01-17 14:36:17.8199 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-17 14:36:17.8199 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT -156558308 , 2730927, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = -156558308 AND [user_id] = 2730927) 

2018-01-17 14:36:46.0365 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-17 14:36:46.0365 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:496 user_id:459569 chat_id:-156558308 username:Sergeiaf name: Сергей Федоров date: 1516189005 message:'/d' 

2018-01-17 14:36:46.0565 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /d 

2018-01-17 14:36:46.0565 | DeeBeeTeeDB.DBAPI.Command_details | DEBUG | Поиск пользователя 'Sergeiaf' 

2018-01-17 14:36:46.0725 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Sergeiaf' 

2018-01-17 14:36:46.1514 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Sergeiaf'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Sergeiaf'), 0) as Bal  

2018-01-17 14:36:46.2355 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | GetUserDetailsSQL: SELECT to_user, SUM(amount) as amount FROM (SELECT [tid], [from_user], [amount], [to_user] FROM [dbo].[transactions] union SELECT [tid], [to_user], [amount]*(-1), [from_user] FROM [dbo].[transactions] ) AS T WHERE from_user = 'Sergeiaf' GROUP BY to_user 

2018-01-17 14:36:46.3204 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Sergeiaf to: Infom amount: 169,00  

2018-01-17 14:36:46.3204 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Sergeiaf to: TretyakRoman amount: 355,00  

2018-01-17 14:36:46.3394 | DeeBeeTeeDB.DBAPI.Command_details | INFO | Возврат результата команды details с сообщениемДетали пользователя @Sergeiaf. Баланс: 524,00
 @Infom должен 169,00 @Sergeiaf
 @TretyakRoman должен 355,00 @Sergeiaf
 

2018-01-17 14:36:46.5546 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-17 14:36:46.5546 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT -156558308 ,'group' ,'ОФФшор', 'Sergeiaf' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = -156558308) 

2018-01-17 14:36:46.6485 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-17 14:36:46.6485 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT -156558308 , 459569, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = -156558308 AND [user_id] = 459569) 

2018-01-17 14:36:49.0531 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9360 

2018-01-17 14:37:53.5550 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9420 

2018-01-17 14:38:58.0105 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9480 

2018-01-17 14:40:02.5188 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9540 

2018-01-17 14:41:12.8425 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9600 

2018-01-17 14:42:22.9179 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9660 

2018-01-17 14:43:45.9111 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9720 

2018-01-17 14:44:10.8987 | DeeBeeTeeAlphaBot.Program.Main | INFO | Запуск бота 

2018-01-17 14:44:10.9392 | DeeBeeTeeAlphaBot.Program.Main | INFO | Подключение к БД 

2018-01-17 14:44:11.5026 | DeeBeeTeeAlphaBot.Program.Main | INFO | Запуск пуллинга telegram API 

2018-01-17 14:45:18.5852 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 60 

2018-01-17 14:46:23.8048 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 120 

2018-01-17 14:47:28.3503 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 180 

2018-01-17 14:48:34.4576 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 240 

2018-01-17 14:49:40.5582 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 300 

2018-01-17 14:50:47.4061 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 360 

2018-01-17 14:51:52.4586 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 420 

2018-01-17 14:52:57.2994 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 480 

2018-01-17 14:54:08.0081 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 540 

2018-01-17 14:55:12.8962 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 600 

2018-01-17 14:56:17.5463 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 660 

2018-01-17 14:57:22.3916 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 720 

2018-01-17 14:58:29.1083 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 780 

2018-01-17 14:59:34.0083 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 840 

2018-01-17 15:00:39.2819 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 900 

2018-01-17 15:01:44.6958 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 960 

2018-01-17 15:02:52.0897 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1020 

2018-01-17 15:04:00.1891 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1080 

2018-01-17 15:05:05.8695 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1140 

2018-01-17 15:06:12.8985 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1200 

2018-01-17 15:07:17.6430 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1260 

2018-01-17 15:08:23.0460 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1320 

2018-01-17 15:09:28.5140 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1380 

2018-01-17 15:10:34.4186 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1440 

2018-01-17 15:11:41.1265 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1500 

2018-01-17 15:12:51.6672 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1560 

2018-01-17 15:13:58.6490 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1620 

2018-01-17 15:15:04.8102 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1680 

2018-01-17 15:16:12.3872 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1740 

2018-01-17 15:17:18.8485 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1800 

2018-01-17 15:18:24.3256 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1860 

2018-01-17 15:19:30.8891 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1920 

2018-01-17 15:20:35.9014 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1980 

2018-01-17 15:21:45.4463 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2040 

2018-01-17 15:22:50.5603 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2100 

2018-01-17 15:23:57.8890 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2160 

2018-01-17 15:25:02.6075 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2220 

2018-01-17 15:26:06.9337 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2280 

2018-01-17 15:27:11.6969 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2340 

2018-01-17 15:28:16.0851 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2400 

2018-01-17 15:29:26.0242 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2460 

2018-01-17 15:30:32.0099 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2520 

2018-01-17 15:31:36.8070 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2580 

2018-01-17 15:32:42.3986 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2640 

2018-01-17 15:33:46.8213 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2700 

2018-01-17 15:34:53.2811 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2760 

2018-01-17 15:35:59.6872 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2820 

2018-01-17 15:37:04.0310 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2880 

2018-01-17 15:38:09.7462 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2940 

2018-01-17 15:39:17.3501 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3000 

2018-01-17 15:40:24.7563 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3060 

2018-01-17 15:41:32.3443 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3120 

2018-01-17 15:42:41.8200 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3180 

2018-01-17 15:44:01.5439 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3240 

2018-01-17 15:45:06.6790 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3300 

2018-01-17 15:46:11.2092 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3360 

2018-01-17 15:47:17.1390 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3420 

2018-01-17 15:48:21.4772 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3480 

2018-01-17 15:49:29.4957 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3540 

2018-01-17 15:50:33.8948 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3600 

2018-01-17 15:51:44.8400 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3660 

2018-01-17 15:52:51.5982 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3720 

2018-01-17 15:54:00.4683 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3780 

2018-01-17 15:55:10.2649 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3840 

2018-01-17 15:56:18.6204 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3900 

2018-01-17 15:57:29.5102 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3960 

2018-01-17 15:58:42.5833 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4020 

2018-01-17 15:59:50.2297 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4080 

2018-01-17 16:01:03.4276 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4140 

2018-01-17 16:02:09.1657 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4200 

2018-01-17 16:03:15.3586 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4260 

2018-01-17 16:04:22.4584 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4320 

2018-01-17 16:05:32.0655 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4380 

2018-01-17 16:06:36.6757 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4440 

2018-01-17 16:07:41.1183 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4500 

2018-01-17 16:08:45.6485 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4560 

2018-01-17 16:09:54.7292 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4620 

2018-01-17 16:10:59.0883 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4680 

2018-01-17 16:12:03.5561 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4740 

2018-01-17 16:13:10.8996 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4800 

2018-01-17 16:14:16.9514 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4860 

2018-01-17 16:15:22.5254 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4920 

2018-01-17 16:16:28.4509 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4980 

2018-01-17 16:17:36.4260 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5040 

2018-01-17 16:18:45.4203 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5100 

2018-01-17 16:19:50.0117 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5160 

2018-01-17 16:21:02.2439 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5220 

2018-01-17 16:22:06.6723 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5280 

2018-01-17 16:23:11.3202 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5340 

2018-01-17 16:24:15.8562 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5400 

2018-01-17 16:25:21.0959 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5460 

2018-01-17 16:26:25.6238 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5520 

2018-01-17 16:27:31.3166 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5580 

2018-01-17 16:28:38.5974 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5640 

2018-01-17 16:29:44.7191 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5700 

2018-01-17 16:30:51.4587 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5760 

2018-01-17 16:31:58.3103 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5820 

2018-01-17 16:33:06.0019 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5880 

2018-01-17 16:34:14.6077 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5940 

2018-01-17 16:35:25.8413 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6000 

2018-01-17 17:46:24.6387 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:24.7358 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:24.7468 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:24.7468 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:24.7668 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:24.7668 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:24.7778 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:24.7778 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:24.7908 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:24.7908 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:24.7908 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:24.8089 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:24.8089 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:24.8089 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6060 

2018-01-17 17:46:24.8339 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:24.8439 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:24.8699 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:25.5191 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:25.5972 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:25.8894 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:25.9384 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:25.9975 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.0135 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.0345 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.0475 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.0615 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.0735 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.0735 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.0895 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.1035 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.1035 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.1185 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.1546 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.1666 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.1666 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.1856 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.1956 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.2216 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.2296 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.2506 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.2596 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.2596 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.2596 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.2746 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.2746 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.2746 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.2896 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.2896 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.2896 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.3057 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.3057 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.3057 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.3207 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.3207 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.3207 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.3357 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.3357 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.3357 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.3357 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.3547 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.3547 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.3547 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.3707 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.3707 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.3827 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.3827 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.3827 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.3827 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.4347 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.4488 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.4488 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.4488 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.4638 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.4638 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.4638 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6120 

2018-01-17 17:46:26.4778 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.4778 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.4778 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.4918 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.4918 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.4918 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5068 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5068 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5068 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5068 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5258 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5258 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5258 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5408 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5408 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5408 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5548 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5548 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5548 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5698 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5698 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5698 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5848 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5848 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5848 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.5848 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6039 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6039 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6039 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6179 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6179 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6179 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6329 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6329 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6329 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6329 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6519 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6519 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6519 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6669 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6669 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6669 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6819 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6819 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6819 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.6959 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.7379 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.7429 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.7429 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.7580 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.7580 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.7810 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.7890 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.7890 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.8070 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.8470 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.8700 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.8700 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.8820 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.8820 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.8820 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6180 

2018-01-17 17:46:26.9051 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.9131 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.9131 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.9131 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.9331 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.9331 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.9451 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.9451 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.9451 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.9631 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.9631 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.9631 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.9871 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.9951 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:26.9951 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.0291 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.0401 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.0401 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.0622 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.0712 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.1092 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.1182 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.1439 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.1484 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.1484 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.1484 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.1654 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.1654 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.1654 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.1804 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.1804 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.1804 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.1974 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.1974 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.1974 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.2124 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.2124 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.2124 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.2284 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.2284 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.2284 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.2464 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.2464 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.2574 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.2574 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.2759 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.2759 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.2759 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.2920 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.2920 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.3095 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.3095 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.3205 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.3205 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.3205 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.3385 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.3385 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.3385 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.3551 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.3551 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.3551 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6240 

2018-01-17 17:46:27.3701 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.3701 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.3831 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.3831 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.3831 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.4011 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.4011 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.4011 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.4191 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.4191 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.4301 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.4301 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.4461 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.4461 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.4461 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.4622 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.4622 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.4622 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.4802 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.4802 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.4802 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.4982 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.4982 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.5092 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.5092 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.5092 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.5272 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.5272 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.5392 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.5392 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.5392 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.5392 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.5592 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.5592 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.5712 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.5712 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.5712 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.5862 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.5862 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.5862 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.6032 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.6032 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.6032 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.6203 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.6203 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.6323 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.6323 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.6323 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.6503 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.6503 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.6503 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.6673 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.6673 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.6803 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.6803 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.6803 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.7003 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.7003 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.7123 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.7123 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.7123 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6300 

2018-01-17 17:46:27.7363 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.7443 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.7443 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.7443 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.7634 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.7634 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.7764 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.7764 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.7764 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.7924 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.7924 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.7924 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.8094 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.8094 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.8204 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.8204 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.8204 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.8364 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.8364 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.8364 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.8534 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.8534 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.8764 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.8824 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.8824 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.8824 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.9024 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.9024 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.9135 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.9135 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.9135 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.9315 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.9315 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.9315 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.9485 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.9485 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.9605 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.9605 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.9605 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.9755 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.9755 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.9755 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.9925 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.9925 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:27.9925 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.0085 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.0085 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.0085 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.0245 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.0245 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.0245 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.0395 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.0395 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.0535 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.0535 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.0535 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.0706 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.0706 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.0706 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.0856 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.0856 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6360 

2018-01-17 17:46:28.0856 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1026 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1026 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1026 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1176 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1176 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1176 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1376 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1376 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1476 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1476 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1476 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1476 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1686 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1686 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1786 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1786 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1786 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1956 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1956 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.1956 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.2136 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.2136 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.2257 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.2257 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.2257 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.2417 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.2417 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.2417 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.2567 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.2567 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.2567 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.2567 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.2767 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.2767 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.2767 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.2907 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.2907 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.2907 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3057 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3057 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3057 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3207 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3207 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3207 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3357 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3357 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3357 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3507 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3507 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3507 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3507 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3698 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3698 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3698 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3848 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3848 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3848 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3998 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3998 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.3998 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6420 

2018-01-17 17:46:28.4148 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.4358 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.4358 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.4508 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.4508 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.4798 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.4798 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.4798 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.4998 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.5098 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.5098 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.5279 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.5279 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:46:28.5519 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-17 17:47:21.2054 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6480 

2018-01-17 17:48:29.9899 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6540 

